Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -1233,14 +1232,13 @@ public void downloadDocket(List<Process> processes) throws IOException {
             File file = module.generateMultipleDockets(serviceManager.getProcessService().getDocketData(processes),
                     xsltFile);
 
-            writeToOutputStream(facesContext, file);
-            facesContext.responseComplete();
+            writeToOutputStream(facesContext, file, "batch_docket.pdf");
         }
     }
 
-    private void writeToOutputStream(FacesContext facesContext, File file) throws IOException {
+    private void writeToOutputStream(FacesContext facesContext, File file, String filename) throws IOException {
```

Comment:
if you already change it, you could use camelCases and this overwrite of value is not necessary